ariadne/executable_schema.py

```diff
@@ -1,21 +1,23 @@
-from typing import List, Union
+from typing import Dict, List, Type, Union
 
 from graphql import (
     DocumentNode,
     GraphQLSchema,
     build_ast_schema,
     extend_schema,
-    validate_schema,
     parse,
+    validate_schema,
 )
 
 from .enums import set_default_enum_values_on_schema
+from .schema_visitor import SchemaDirectiveVisitor
 from .types import SchemaBindable
 
 
 def make_executable_schema(
     type_defs: Union[str, List[str]],
     bindables: Union[SchemaBindable, List[SchemaBindable], None] = None,
+    directives: Dict[str, Type[SchemaDirectiveVisitor]] = None,
```

Comment:
Let's make directives option explicit:

```suggestion
    *,
    directives: Dict[str, Type[SchemaDirectiveVisitor]] = None,
```